index.html

```diff
@@ -3761,8 +3762,8 @@ <h3>The <dfn>ThingFilter</dfn> dictionary</h3>
       </p>
       <pre class="idl">
         dictionary ThingFilter {
-          DiscoveryMethod method = "directory";
-          USVString? url;
+          DiscoveryMethod method = "direct";
+          required USVString url;
```

Comment:
About this, let's discuss in the call.

Reply:
Scripting call 2022-09-26
* Move out this change in another PR, keep this PR a syntactic change 
* `auto` method ?

Reply:
Moved these lines to a new PR #434 now :)